frontend/src/httpLayers/login.http.js

```diff
@@ -1,10 +1,21 @@
 import wofApi from "./wofApi.js";
 
+/**
+ * Authenticates the user to the backend service of
+ * WikiOnFire. The response from the server sets
+ * the authentication and authorization `httpOnly` cookie.
+ *
+ * @param {String} username
+ * @param {String} password
+ * @returns {Object} the user object with the following fields:
+ *   id, username, password, email, account_type, account_status, createdAt,
```

Comment:
We should not return password here

Reply:
Mhm, I am going to have a look at the endpoint as well. It shouldn't return password as well.

Reply:
Fixed